configure.ac

```diff
@@ -361,6 +361,22 @@ case $host in
    *linux*)
      TARGET_OS=linux
      ;;
+   *dragonfly*)
```

Comment:
These aren't required for Mac; I did a bunch of build fixed in BU around the time and pulled 2 in.  If you want me to separate these I can.